front-end/src/components/Body/Current/Current.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "./style.scss";
+
+const Current = (props) => {
```

Comment:
@beeeeesankun -san
The code use ES6 function is good
We can use functional in javascript is better than ES6. Because the code run and combine faster.
Thanks